### PR TITLE
Added in the code for the TurnMaster class

### PR DESCRIPTION
**Pull request by Snapshot0010**
I added in the implementation for the TurnMaster class. I did this by adding in two additional methods to check for the status of the player's turns and resetting the player's turns. Whenever all players have finished their turn or after a certain amount of time (currently 30 seconds but can be easily changed)  it increments the currTurn variable and resets the isDone variable in all player objects. I also fixed some of the code in PlayerScripts. There appeared to be a missing semicolon and a couple of methods that should have void as their return type did that.

### CODE REVIEW

File: Crypto Wars/Assets/Scripts/TurnMaster.cs [hunk 1]
```diff
@@ -5,19 +5,45 @@
...
     {
         currTurn = 0;
+        turnTimer = 0f;
     }
 
     // Update is called once per frame
     void Update()
     {
-        //Test to see if either of the two game end scenarios have been met
+        if(allAreDone(Players) || turnTimer == maxTurnTime)
```
[the remark this answers is not in the document]
Comment (CodingAdam) [reply Crypto Wars/Assets/Scripts/TurnMaster.cs:21]: I would also recommend doing that, as having an exact time could cause some edge cases. Mainly if the update frames are slightly out of sync. It's a good practice for values that are iterating in the background.

[the remark this answers is not in the document]
Comment (Snapshot0010) [reply Crypto Wars/Assets/Scripts/TurnMaster.cs:21]: Alright, I'll do that whenever I push another change. For now let's accept this one unless there are other things that need to be changed

[the remark this answers is not in the document]
Comment (CodingAdam) [reply Crypto Wars/Assets/Scripts/TurnMaster.cs:21]: Fair enough I'll approve the pr, I'll put a reminder down.

**Review by CodingAdam — DISMISSED** [review]
Approving, we need some of the issues fixed with playerscript and the turn timer thing can be fixed easily in future prs.

**Review by Grimmblood — APPROVED** [review]

**Pull request merged**